src/components/JobVariableOverridesInput.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <JsonInput
+    v-model="modelValue"
+    show-line-numbers
+    :min-lines="3"
+    class="job-variable-overrides-input"
+    show-format-button
+  />
+</template>
+
+<script setup lang="ts">
+  import JsonInput from '@/components/JsonInput.vue'
+
+  const modelValue = defineModel<string>()
+</script>
+
+<style>
+.job-variable-overrides-input { @apply
+  min-h-[2.5rem]
+  min-w-full
```

Comment:
I ported these styles over from the `<DeploymentForm />` tho I can't... for the life of me figure out what these min-h and min-w were for? 

if you know - i'd love to know 🙃

Reply:
Otherwise when you resize you can get into a weird spot.

Reply:
This component is basically what the default should be for a json input. Someday we should fix that lol

Reply:
Ohhhh I think I did my rem math wrong - I see the min-h stopping you from resizing down too small now. 

still not that sure about the min-w-full - feels like it was maybe an artifact from being in some flex container where it wasn't stretching but not sure

Reply:
`min-w-full` might not be needed. Could be leftover from using `resize` initially.